src/ui/public/chrome/Tab.js

```diff
@@ -28,7 +29,16 @@ export default class Tab {
 
     this.lastUrlStoreKey = `lastUrl:${this.id}`;
     this.lastUrlStore = spec.lastUrlStore;
-    this.lastUrl = this.lastUrlStore ? this.lastUrlStore.getItem(this.lastUrlStoreKey) : null;
+
+    this.lastUrl = null;
+    if (this.lastUrlStore) {
+      this.lastUrl = this.lastUrlStore.getItem(this.lastUrlStoreKey);
+      if (this.lastUrl && !this.lastUrl.startsWith(this.rootUrl)) {
```

Comment:
yep, that is very true but acceptable IMO. This is why line 37 and 67 are no longer throw an error. Open to better ways of handling this though.